extend wmts cascaded layer doc (#5698)
geoserver/geoserver: doc/en/user/source/data/cascaded/wmts.rst

* GetLegendGraphic requests aren't supported.

* Image format cannot be specified.  GeoServer will attempt to request PNG images, and if that fails will use the remote server's default image format.

> Images output discrepancies in a cascade WMTS Layer

Reviewer: ```suggestion
Images output discrepancies in a cascaded WMTS Layer
```

---------------------------------------------------

> WMTS it is a service that serves tiles and they have been generated for a concrete resolution/scale denominator, so asking a WMTS cascaded layer to generate WMS getMap images or other WMTS tiles with other scale denominators will generate discrepancies on the image outpus.

Reviewer: ```suggestion
WMTS it is a service that serves tiles and they have been generated for a concrete resolution/scale denominator, so asking a WMTS cascaded layer to generate WMS getMap images or other WMTS tiles with other scale denominators will generate discrepancies on the image outputs.
```

* If the image is streched (scaled out) and the difference is notable, the borders, lines, and texts that appear in it could be blurring.

* In the other hand if shrinked, the same object and shape could appear smaller than the original size and not display properly as they were intended to be displayed for that scale.
  
.. figure:: images/cascaded_wmts.png
  :align: center
  

>   *This figure compares a result image from a WMTS to a cascaded layer which has been slightly streched or scaled out*

Reviewer: ```suggestion
  *This figure compares the resulting image from a WMTS to a cascaded layer which has been slightly streched or scaled out*
```

  *Left image shows a original wmts layer at its defined zoom level 4 which scale denominator is about 4M*
  *Right image whows a cascaded wmts layer as wms layer with at different scale denominator (the closest to its homologous cascaded layer) which is about 5M*

> It should be taken into account this kind of performace if requesting or republishing in different scale denominators.

Author: It is like a warning to say that if wmts cascaded layer is published (for wmts) or visualised (for wms) in other scale denominators than the original service, it would result in those image issues.

> It should be taken into account this kind of performace if requesting or republishing in different scale denominators.

Author: I can replace it to: _It should be taken into account that if a wmts cascaded layer is published (for wmts) or visualised (for wms) in other scale denominators than the original service, it would result in those image issues._
But you can suggest a better or more understandable sentence as well, not a problem.

> It should be taken into account this kind of performace if requesting or republishing in different scale denominators.

Reviewer 2: Or it can simply be removed, as it repeats what was already said above. Taking care of it.
